packages/webpack/src/config/base.js

```diff
@@ -397,6 +397,15 @@ export default class WebpackBaseConfig {
     }
 
     // Clone deep avoid leaking config between Client and Server
-    return this.extendConfig(cloneDeep(config))
+    const extendedConfig = this.extendConfig(cloneDeep(config))
+    const optimization = extendedConfig.optimization
+    if (optimization && optimization.minimizer && extendedConfig.devtool) {
+      const terser = optimization.minimizer.find(p => p instanceof TerserWebpackPlugin)
```

Comment:
General tip: Terser is not the only thing that is affected. All loaders for css also need an update. I suggest adding a comment in above of this condition to remove this in Nuxt 3.x.

Reply:
Not quite following :) for css we have cssSourceMap property.  How do u want to handle it in nuxt 3 ? Something like devtool: { server: 'server-sourcemap', client: 'client-source-map' }  ?

Reply:
Yes. I think it would be better to explicitly set devTool options and not using `extend`. Or better [SourceMapDevToolPlugin](https://webpack.js.org/plugins/source-map-dev-tool-plugin) for more control.